backend/config/curation-api.yml

```diff
@@ -43,8 +43,7 @@ paths:
       summary: Fetch Collections metadata.
       description: Fetch all Collections metadata. Authorization required for private collections.
       security:
-        - cxguserCookie: []
-        - curatorAccess: []
+        - curatorAccessLenient: []
```

Comment:
we need to let all requests through but we check with user before adding private collections to response, right